clusterloader2/pkg/framework/framework.go

```diff
@@ -60,6 +60,16 @@ func NewFramework(path string) (*Framework, error) {
 	return &Framework{0, clientSet, dynamicClient}, nil
 }
 
+// GetClientSet returns clientSet client.
+func (f *Framework) GetClientSet() clientset.Interface {
+	return f.clientSet
+}
+
+// GetDynamicClient returns dynamic client.
+func (f *Framework) GetDynamicClient() dynamic.Interface {
```

Comment:
This is not called anywhere. Why have it then?

Reply:
I thought that if there is need for getter to ClientSet than at some point will be need for DynamicClient also. It's just a getter.

Reply:
If we need it later, then let's add later. Otherwise this may potentially lie around as dead code forever.

Reply:
done.